apis/keda/v1alpha1/cloudevent_types.go

```diff
@@ -0,0 +1,69 @@
+/*
+Copyright 2021 The KEDA Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+// CloudEvent defines how a trigger can authenticate globally
+// +genclient
+// +kubebuilder:resource:path=cloudevents,scope=Namespaced
+// +kubebuilder:subresource:status
+type CloudEvent struct {
+	metav1.TypeMeta   `json:",inline"`
+	metav1.ObjectMeta `json:"metadata,omitempty"`
+
+	Spec CloudEventSpec `json:"spec"`
+}
```

Comment:
Do we want a ClusterCloudEvent for configuring them at cluster level? Is it useful? (maybe it's not the moment now) @tomkerkhove

Reply:
Yes, that's part of the plan but think out of scope here

Reply:
Yes, sorry.
I reviewed it yesterday and I left some comments but then I read the issue again with the plan and I delete some of them. I guess that I forgot this

Reply:
No worries at all